### DISCUSSION TITLE
Can't figure out how to select child component

### DISCUSSION THREAD

**Discussion by hoclun-rigsep** [noprompt/garden#190] [Q&A] [answered]
Hello. Trying to simulate this selector:

```css
.A > .B
```

I can't quite figure out how to do it.

**Comment by harold** [comment, marked as answer]
I think you're looking for this:

https://github.com/noprompt/garden/blob/e9d987d95e26bd22e181ef4332c801cfe4533870/src/garden/selectors.cljc#L984-L994

And using it looks like this:

```
garden-test.core> (require '[garden.core :refer [css]])
nil
garden-test.core> (css [(garden.selectors/> :.A :.B) {:background :red}])
".A > .B {\n  background: red;\n}"
```

Another thing to know is that you can use strings as an escape hatch, so this renders the same:

```
garden-test.core> (css [".A > .B" {:background :red}])
".A > .B {\n  background: red;\n}"
```

Hope that helps.

**Reply by hoclun-rigsep** [reply]
It does; thank you.
